shuup/front/templates/shuup/front/macros/general.jinja

```diff
@@ -257,9 +257,19 @@
             </div>
             <div class="collapse navbar-collapse" id="admin-tools-menu">
                 <ul class="nav navbar-nav navbar-right">
+                {% set page = shuup.general.get_admin_url() %}
```

Comment:
check the comment above and pass the object explicitly to the function

Reply:
Can you elaborate a bit more on that^?

Reply:
instead of invoking `shuup.some_function()` and expecting the `context` has the object, use an expected parameter like `shuup.some_function(my_object)`

Reply:
run `python` and type: `import this`, check item #2 :nerd_face:

Reply:
Good now? Also why would I use model_url if I'm using the function that's doing the same thing?

Reply:
because it uses `provides`